webpack.config.js

```diff
@@ -2,30 +2,58 @@ const process = require('process');
 const path = require('path');
 const webpack = require('webpack');
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const ThreadsPlugin = require('threads-plugin');
 const nodeExternals = require('webpack-node-externals');
 
 module.exports = {
   entry: './src/index.ts',
   devtool: "source-map",
   output: {
     path: path.resolve(__dirname, 'dist'),
-    filename: 'index.js',
+    filename: 'encryptedfs.js',
     library: 'encryptedfs',
     libraryTarget: 'umd',
   },
+  externals: {
+    virtualfs: "virtualfs",
+    threads: "threads",
+  },
+  devtool: "source-map",
+  devServer: {
```

Comment:
Remove the `devServer` this is not a web application.

Reply:
this was previously used to debug on the console in the browser, but it is replaced with the more appropriate `ts-node`